docs/guides/run/health-checks.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Healthchecks
+description: description
+icon: heart-pulse
+---
+
+## Configuration
+
+Below you can find an example with a toml configuration file for a service with healthcheck enabled.
+
+```toml
```

Comment:
I'd add here only the relevant bits for the healthchecks, for a full example we already have the cli.

Reply:
ok